src/core/build/types.ts

```diff
@@ -217,6 +217,8 @@ declare module "nitro/types" {
         jsx: "preserve",
         jsxFactory: "h",
         jsxFragmentFactory: "Fragment",
+        allowImportingTsExtensions: true,
+        rewriteRelativeImportExtensions: true,
```

Comment:
It is not required for us as nitro does not use tsc compiler. 

allow importing ts extensions is nice (we could also backport it to v2 branch as non breaking change)

Reply:
Which structure do we broadcast from without tsc ?  Which place in the code should I examine. I need to learn this structure